network/p2p/gossip/bloom.go

```diff
@@ -84,7 +100,7 @@ func ResetBloomFilterIfNeeded(
 	if err != nil {
 		return false, err
 	}
-
+	bloomFilter.maxCount = bloom.EstimateCount(numHashes, numEntries, bloomFilter.resetFalsePositiveProbability)
```

Comment:
same comment as in https://github.com/ava-labs/avalanchego/pull/2591/files#r1446119901

Reply:
Responded here: https://github.com/ava-labs/avalanchego/pull/2591#discussion_r1447116178